src/Native/build.sh

```diff
@@ -96,6 +96,12 @@ fi
 
 __cmake_defines="${__cmake_defines} -DVERSION_FILE_PATH:STRING=${__versionSourceFile}"
 
+# This should be done when nuget is published but until then lets keep these below lines.
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    echo "Changing libMklImports.dylib's name to include @rpath."
+    install_name_tool "-id" @rpath/libMklImports.dylib "$__mkllibpath"/libMklImports.dylib
```

Comment:
Similar comment here as before - this is only going to work during tests in our repo. How will it work for customers who are using our NuGet package?

Reply:
Senja will be updating the nuget today with this line. Also, I did mention above this is temp :)

---
In reply to: [221635422](https://github.com/dotnet/machinelearning/pull/977#discussion_r221635422) [](ancestors = 221635422)